node_tree.py

```diff
@@ -307,6 +478,46 @@ def draw_color(self, context, node):
         return self.nodule_color
 
 
+class SvLinkMatrix(bpy.types.Operator):
```

Comment:
seriously. no. don't do this. 

make a SvLinkNewNode, and make it accept a bl_idname and an offset. so you can reuse the same operator for any number of new "New Node"s .  Here the _action_ of adding a new node is more important than the node kind..